api/v1beta1/flinkcluster_validate.go

```diff
@@ -89,6 +94,39 @@ func (v *Validator) ValidateUpdate(old *FlinkCluster, new *FlinkCluster) error {
 	return nil
 }
 
+func (v *Validator) checkControlAnnotations(old *FlinkCluster, new *FlinkCluster) error {
```

Comment:
Can we add a unit test for this function?

Reply:
Done.